spec/models/order_spec.rb

```diff
@@ -0,0 +1,217 @@
+require 'rails_helper'
```

Comment:
For many comments you'll need to do the fix at multiple places, please find yourself and fix this file completely